aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -47,118 +130,124 @@ xxlarge: `${baseSpacing * 64}px`, // 1536px
-    large: `${baseSpacing * 2}px`, // 48
-    xlarge: `${baseSpacing * 4}px`, // 96
-
-}
-```
-
-Given this code snippet, if you were to implement the following code, you would have a Box with width of 384px, height of 192px, and pad of 48px.
+The example below shows `xlarge` padding resolved to 48px:
 
 ```
 <Box
     width="medium"
-    height="small"
-    pad="large"
+    height="xsmall"
+    pad="xlarge"
 />
 ```
 
-<Box width="medium" margin={{bottom: 'large'}}>
+<Box width="medium" margin={{ bottom: 'large' }}>
   <Image
     src="/edge-sizing.png"
     alt="Demonstration of t-shirt sizing for pad on Box"
   />
 </Box>
 
-Border sizing is controlled by `borderSize`.
 
-```
-// base.js in Grommet
+The following snippet produces a Box with width 384px, height 192px, and a 2px border.
 
-borderSize: {
-    xsmall: '1px',
-    small: '2px',
-    medium: `${baseSpacing / 6}px`, // 4
-    large: `${baseSpacing / 2}px`, // 12
-    xlarge: `${baseSpacing}px`, // 24
-}
 ```
-
-Given this code snippet, if you were to implement the following code, you would have a Box with width of 384px, height of 192px, and a border with a width of 2px.
-
-```
-<Box 
-    border={{ color: 'border', size: 'xsmall' }}
+<Box
+    border={{ color: 'border', size: 'small' }}
     width="medium"
-    height="xsmall" 
+    height="xsmall"
 />
 ```
 
-<Box width="medium" margin={{bottom: 'xlarge'}}>
+<Box width="medium" margin={{ bottom: 'xlarge' }}>
   <Image
     src="/border-sizing.png"
     alt="Demonstration of t-shirt sizing for border on Box"
   />
 </Box>
 
-### Composability and scaling between t-shirt sizes
+### Composability
 
-Sizes are intended to help give a UI a unified, harmonious experience. Therefore, there is a relationship between the pixel values that each size value maps to.
-
-A given size (e.g., medium) is one half of the next size up (e.g., large).
+The v7 scale maintains additive and divisional consistency; sums or partitions of smaller tokens align precisely with larger counterparts.
 
 ```
-small (192px) + small (192px) = medium (384px)
-medium (384px) + medium (384px) = large (768px)
+// Container sizes maintain doubling relationships at key intervals:
+xsmall (192px) + xsmall (192px) = medium (384px)
+medium (384px) + medium (384px) = xlarge (768px)
+
+// New intermediate sizes fill gaps:
+small (256px) = medium (384px) - xxsmall (128px)
+large (512px) = medium (384px) + xxsmall (128px)
+
+// Spacing sizes follow similar mathematical relationships:
+xsmall (12px) + xsmall (12px) = medium (24px)
+medium (24px) + medium (24px) = xlarge (48px)
 ```
 
-### Default `size` of "medium"
+Intermediate tokens address previous gaps where `medium` and `large` formed an overly broad jump.
+
+### Composition visualization
+
+The beauty of the Dual Doubling scale is its mathematical precision, allowing sizes to compose and decompose in predictable ways. The visualization below demonstrates how different t-shirt sizes relate to each other:
```

Comment:
In some places we refer to the 'v7 scale' in others we say 'the Dual Doubling scale', thinking we should align to a consistent language.

I'm wondering if we should refer to the v6 and earlier scale as the 'legacy' t-shirt scale and v7 and above is just the t-shirt size scale.